install/install.go

```diff
@@ -617,28 +617,31 @@ func (c *installer) configureAPIServerImage() (image string, args []string, env
 			corev1types.PullNever
 	}
 	if c.debugImages {
-		return imagePrefix + "api-server-debug:latest", []string{
-				"--kubeconfig=",
-				"--authentication-kubeconfig=",
-				"--authorization-kubeconfig=",
-				"--service-name=compose-api",
-				"--service-namespace", c.commonOptions.Namespace,
-			},
+		return imagePrefix + "api-server-debug:latest",
+			c.configureAPIServerArgs(),
 			[]corev1types.EnvVar{},
 			corev1types.PullNever
 	}
 	return imagePrefix + "api-server" + ":" + c.commonOptions.Tag,
-		[]string{
-			"--kubeconfig", "",
-			"--authentication-kubeconfig=",
-			"--authorization-kubeconfig=",
-			"--service-name=compose-api",
-			"--service-namespace", c.commonOptions.Namespace,
-		},
+		c.configureAPIServerArgs(),
 		[]corev1types.EnvVar{},
 		c.commonOptions.PullPolicy
 }
 
+func (c *installer) configureAPIServerArgs() []string {
+	args := []string{
+		"--kubeconfig", "",
+		"--authentication-kubeconfig=",
+		"--authorization-kubeconfig=",
+		"--service-name=compose-api",
+		"--service-namespace", c.commonOptions.Namespace,
+	}
+	if c.commonOptions.SkipLivenessProbes {
```

Comment:
Or you can also have something like this at top level:
```go
if c.commonOptions.SkipLivenessProbes {
    c.commonOptions.HealthzCheckPort = 0
}
```
So here you would only have
```go
return append(args, fmt.Sprintf("--healthz-check-port=%d", c.commonOptions.HealthzCheckPort))
```

Reply:
fixed